CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 ## Unreleased
 
+### Update drug categorization
+- confers_resistance_to() now gets drugs for the whole AMR gene family. For example, OXA-19 previously only returned cephalosporin and penam, but now will also return oxacillin (from AMR gene family).
```

Comment:
Can you encode this example as a test? In particular, as a test that is human readable (as discussed before)

Reply:
I've added seperate tests for both OXA-19 and ant(2")-Ia (the gene for the workflow figure). Are they ok?